views/display.html

```diff
@@ -14,6 +14,37 @@
             </div>
         </div>
 
+        <div id="users">
+            <div id="news">
```

Comment:
do we need to put each news card inside a `#news ul#newslist`? I think we could probably just make each card the exact same way as we do the users, just with a div and then the same two divs you have currently